src/applications/discharge-wizard/helpers/index.jsx

```diff
@@ -263,6 +263,12 @@ export const deriveIsAirForceAFRBAPortal = formValues =>
 
 // v2 Helpers
 
+export const determineBranchOfService = key =>
```

Comment:
Why would the Marines key map to Navy?

Reply:
I had to google this but apparently they do not have a discharge review board and use the Navy one.

Reply:
Oh ok - this would be helpful context for some function documentation :)